src/large_pages/node_large_page.cc

```diff
@@ -282,20 +294,32 @@ bool IsSuperPagesEnabled() {
 }
 #endif
 
+// Functions in this class must always be inlined because they must end up in
+// the `lpstub` section rather than the `.text` section.
+class MemoryMapPointer {
+ public:
+  FORCE_INLINE MemoryMapPointer(size_t size): size_(size) {}
```

Comment:
```suggestion
  FORCE_INLINE explicit MemoryMapPointer(size_t size) : size_(size) {}
```

Reply:
@bnoordhuis I'll move the call into `Reset()`. That would make sense, since you shouldn't be able to `Reset()` to any other kind of pointer into the class. Then we can leave the constructor without parameters, too, because the `size_` and the `mem_` will both be the result of calling `Reset()`.